scribus/ui/prefs_hyphenator.cpp

```diff
@@ -33,7 +33,7 @@ Prefs_Hyphenator::Prefs_Hyphenator(QWidget* parent, ScribusDoc* doc)
 	setupUi(this);
 
 	QStringList languageList;
-	LanguageManager::instance()->fillInstalledHyphStringList(&languageList);
```

Comment:
Why are you changing the hyphenation preferences? We shouldn’t need any changes here.